tooling/hibernate-enhance-maven-plugin/src/main/java/org/hibernate/orm/tooling/maven/MavenEnhancePlugin.java

```diff
@@ -157,7 +157,7 @@ public boolean doExtendedEnhancement(UnloadedClass classDescriptor) {
 
 			writeOutEnhancedClass( enhancedBytecode, file );
 
-			getLog().info( "Successfully enhanced class [" + file + "]" );
```

Comment:
could you change this to make sure the strings aren't appended when the logger is not set to debug?

Something like

    log = getLog()
   if (log.isDebugEnabled()) { log.debug( .... ) }

Reply:
I will make the change and submit another PR.